xmipp

```diff
@@ -1755,13 +1756,22 @@ if __name__ == '__main__':
         else:
             print(red("\nSome error occurred during the compilation\n"))
             sys.exit(1)
-    elif mode=="compileAndInstall":
-        if n>2:
-            Nproc=sys.argv[2]
-        else:
-            Nproc=8
-        compile(Nproc)
-        install("build")
+    elif mode == "compileAndInstall":
+        Nproc = 8
+        dir = "build"
+        if n > 2:
+            for arg in sys.argv[2:]:
```

Comment:
Process all argument except the first two is probably not something we want to do.

Reply:
Yes, we want.

argv[0] = 'xmipp'
argv[1] = 'compileAndInstall'
argv[2:] = <options to process>

The first two are known in this scope.

Reply:
Yes, but the rest can contain unspecified number of other arguments in whichever order.

Reply:
Yes, for that reason there is a loop and a type assertion. In this mode only two arguments are allowed (both optional), one is a pure string (target dir) and the other is a digit string.

I don't get you.